src/uct/api/uct.h

```diff
@@ -417,6 +417,10 @@ typedef enum uct_atomic_op {
 #define UCT_IFACE_FLAG_TAG_EAGER_BCOPY UCS_BIT(51) /**< Hardware tag matching bcopy eager support */
 #define UCT_IFACE_FLAG_TAG_EAGER_ZCOPY UCS_BIT(52) /**< Hardware tag matching zcopy eager support */
 #define UCT_IFACE_FLAG_TAG_RNDV_ZCOPY  UCS_BIT(53) /**< Hardware tag matching rendezvous zcopy support */
+
+#define UCT_IFACE_FLAG_FLUSH_CHECK     UCS_BIT(60) /**< Flush check operation */
```

Comment:
BTW, does it define flag for iface or ep flush? If the latter should we include ep in the name?

Reply:
UCT_IFACE_FLAG_EP_FLUSH_CHECK    ?

Reply:
renamed